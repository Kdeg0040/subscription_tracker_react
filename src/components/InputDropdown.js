import React, { Component } from 'react';

class InputDropdown extends Component {
  render () {
    return (
      <div className={"dropdown is-pulled-left " + this.props.show()}>
        <div className="dropdown-menu">
          <div className="dropdown-content"> 
            {
              this.props.formData.suggestions.map((sugg, ind) => {
                return (
                  <a href="/#" onClick={() => this.props.select(sugg)} className="dropdown-item" key={ind}>
                    <div className="level">
                      <div className="level-left">
                        <div className="level-item">
                          <figure className="image is-32x32">
                            <img src={sugg.logo} alt=""></img>
                          </figure>
                        </div>
                        <div className="level-item">
                        {sugg.name}
                        </div>
                      </div>
                    </div>
                  </a>
                )
              })
            }
          </div>
        </div>
      </div>
    )
  }
}

export default InputDropdown;