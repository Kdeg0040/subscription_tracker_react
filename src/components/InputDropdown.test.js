import React from 'react';
import { shallow } from 'enzyme';

import InputDropdown from './InputDropdown';

const inputDropdown = shallow(<InputDropdown />);

describe('<InputDropdown', () => {
  it('renders correctly', () => {
    expect(inputDropdown).toMatchSnapshot();
  });
});