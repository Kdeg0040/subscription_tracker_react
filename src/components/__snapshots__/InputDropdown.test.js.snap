// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputDropdown renders correctly 1`] = `ShallowWrapper {}`;
